models/articles.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+function retrieveArticleById(id) {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [id])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Not Foun" });
+      }
+
+      return rows;
```

Comment:
Small suggestion, just return rows[0] here, you will always get one article back but rows is always an array. May as well just return that first value here.